src/StokesFSG.jl

```diff
@@ -351,13 +351,13 @@ function Numbering!(N, type, nc)
 
     # Loop through inner nodes of the mesh
     for j=2:size(type.Vx[1],2)-1, i=2:size(type.Vx[1],1)-1
-        if type.Vx[1][i,j] == :Dir_conf || (type.Vx[1][i,j] == :periodic && i==size(type.Vx[1],1)-1)
+        if type.Vx[1][i,j] == :Dirichlet_normal || (type.Vx[1][i,j] == :periodic && i==size(type.Vx[1],1)-1)
```

Comment:
```suggestion
        if type.Vx[1][i,j] === :Dirichlet_normal || (type.Vx[1][i,j] === :periodic && i==size(type.Vx[1],1)-1)
```
triple equality is better for symbols and strings